products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,114 @@
 # Global Documentation Search
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-01-23
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., DevOps
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Search multiple documentation sources from a single, public entrypoint.
+
+### Background
+
+The documentation we have is spread across many different locations (public repos, private repos, GitHub pages, handbooks, etc). There currently is not a way for VFS/VSP users to search all of the documentation sources with a single query.
+
+### High Level Design
+
+We are adding a public landing page that will contain a form where users can search multiple, public documentation sources with a single query.
+
+## Specifics
+
+### Detailed Design
+
+We are adding a public landing page (on a custom domain, if possible) that will contain a search input. That search input will query multiple, public documentation sources with one query.
+
+A successful implementation will have these components.
+
+1. Page with Input Textbox - We need a public page with a search input where users can input their search term(s).
+1. Index/Database - We need a place to store the records that will be returned to the user.
+1. Crawler/Scraper - We need a crawler to frequently scan our documentation sources for additions/deletions/modifications.
+
+We do not currently have an index of our target documentation sources. And we do not currently have infrastructure to crawl our target documentation sources to build said index.
+
+Algolia appears to be the leader when it comes to searching documentation.
+
+### Code Location
+
+_The path of the source code in the repository._
```

Comment:
Re: new repo conversations — _does it_ make most sense to put the code in a new repo? What code? If it's the code for the search tool itself (which is the subject of this design doc, so I assume yes), then yes — or rather, in that case I'm inclined to put the code wherever y'all as engineers think is best. 

But we should also keep in mind that this landing page will host content beyond the search functionality (beyond the scope of this design doc) — it will host the entire documentation homepage, including a table of contents, and longer term may we recommend building out "top docs" as part of that website experience as well. The homepage's table of contents, and the contents of those "top docs" should likely be served from `va.gov-team`. 

In which case, should the search tool code live in a new repo, and the homepage itself ultimately pull content from both the search tool repo and the `va.gov-team` repo? Or, some other arrangement?

Reply:
Just from a separation of concerns point-of-view, I think it probably makes sense to have a repo dedicated to the doc search + future features to build on top of that.